### DISCUSSION TITLE
i want the website starts with dark mode by default

### DISCUSSION THREAD

**Discussion by FoggY9** [smaranjitghose/awesome-portfolio-websites#1144] [Q&A] [answered]
it always starts with light mode. I want it to start with dark by default,
I've tried to do that but the button works the exact opposite 

how do i do that?

**Comment by anushbhatia** [comment, marked as answer]
Set the theme in the sheets from light to dark.
Refer [#1158](https://github.com/smaranjitghose/awesome-portfolio-websites/pull/1158)
